app_dart/app.yaml

```diff
@@ -4,7 +4,12 @@
 
 runtime: custom
 env: flex
-service: default
+readiness_check:
```

Comment:
Could you please share how Dart does this?

Reply:
https://dart.googlesource.com/dart_ci/+/refs/heads/master/appengine/app.yaml

Reply:
Thanks.